stripe-buffer.el

```diff
@@ -136,9 +174,25 @@ Used by `stripe-table-mode' Only the first matching group will be painted."
                                     (goto-char end)
                                     (point))))))
                       ( overlay (apply get-overlay-create-function stripe-region)))
-                 (overlay-put overlay 'face stripe-highlight-face)
-                 (overlay-put overlay 'is-stripe t)
-                 (push overlay sb/overlays)))))
+
+                 (cond
+                  ((unless (require 'dired-subtree nil t)
```

Comment:
AFAIK, `require` will try to find the package in the filesystem, if it's not loaded. Which is not an option in a loop.

You are using `dired-subtree--get-ov` anyway. Should upstream change `fboundp` is more likely to fail silently -- which I think is more desirable.